src/main/java/com/atlassian/bitbucket/jenkins/internal/config/BitbucketPluginConfiguration.java

```diff
@@ -31,11 +32,18 @@ public BitbucketPluginConfiguration() {
     @Override
     public boolean configure(StaplerRequest req, JSONObject json) throws FormException {
         req.bindJSON(this, json);
-        if (getServerList().stream().allMatch(server -> server.validate())) {
+        List<FormValidation> aggregate = new ArrayList<>();
```

Comment:
So to explain this a little bit:
`FormValidation` has no public constructor. It does have an 'aggregate', which appends form messages together and takes the worst `Kind`, but it also injects a lot of HTML that is not rendered when the `FormException` is caught and rendered.
That said I'm not happy with my implementation of the list aggregation, I think it can be made less cumbersome.